var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -425,21 +422,16 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("matio", when="+exodus")
     depends_on("metis", when="+zoltan")
     depends_on("mpi", when="+mpi")
-    depends_on("mpi", when="@15: +python")
-    depends_on("mpi", when="+pytrilinos2")
+    depends_on("mpi", when="+python")
     depends_on("netcdf-c", when="+exodus")
     depends_on("parallel-netcdf", when="+exodus+mpi")
     depends_on("parmetis", when="+mpi +zoltan")
     depends_on("parmetis", when="+scorec")
     depends_on("py-mpi4py", when="+mpi+python", type=("build", "run"))
-    depends_on("py-mpi4py", when="+pytrilinos2", type=("build", "run"))
+    depends_on("py-mpi4py", when="+python", type=("build", "run"))
```

Comment:
This one can stay since PyTrilinos + PyTrilinos2 both need this depends_on()

Reply:
Actually, the line above could be deleted as well:
```
depends_on("py-mpi4py", when="+mpi+python", type=("build", "run"))
```
as we have:
```
depends_on("py-mpi4py", when="+python", type=("build", "run"))
```